examples/sample.rs

```diff
@@ -0,0 +1,77 @@
+use std::path::PathBuf;
+
+use unicorn_engine::{Arch, Mode, Prot, RegisterX86, Unicorn};
+use unicornafl::{afl_fuzz, executor::UnicornFuzzData};
+
+fn place_input_cb<'a>(
+    uc: &mut Unicorn<'a, UnicornFuzzData<bool>>,
+    input: &[u8],
+    _persistent_round: u64,
+) -> bool {
+    let do_x86_64 = uc.get_data().user_data;
+    if do_x86_64 {
```

Comment:
The example needs a comment (i.e., why do we have this here?)
Maybe instead of `do_...` just name it `64bit_mode`?